mct_quantizers/keras/quantize_wrapper.py

```diff
@@ -199,13 +199,15 @@ def get_config(self):
 
             """
             base_config = super(KerasQuantizationWrapper, self).get_config()
-            config = {WEIGHTS_QUANTIZERS: {k: keras.utils.serialize_keras_object(v) for k, v in self.weights_quantizers.items()},
-                      WEIGHTS_VALUES: {k: self.serialize_fn(v) for k, v in self.weight_values.items()}}
-
+            config = {WEIGHTS_QUANTIZERS: {k: keras.utils.serialize_keras_object(v) for k, v in self.weights_quantizers.items()}}
+            # Only create the wrapper attributes that handle positional weights if they exist, so the wrapper is forward
+            # compatible with older MCTQ versions (at least until MCT with start quantizing positional weights)
```

Comment:
typo (with --> will)
and maybe if it is something that we will need to update once enabling positional weights quantization then make it a todo?

Reply:
there's no need to change it when MCT will quantize constants, but it won't be forward compatible for these models anymore.